### ISSUE TITLE
Jupyter UI form default values not reflecting changes from jupyter-web-app-config configMap

### ISSUE THREAD

**Issue by alexchrisramos** [kubeflow/kubeflow#6741]
/kind bug

**What steps did you take and what happened:**
We are looking into changing the default values for Jupyter UI. So that when users creates a new notebook server, the new defaults would take effect.
So we have modified the configMap jupyter-web-app-config in kubeflow namespace.
In the following example, we have changed the default values for the workspaceVolume.
from 10Gi to 20Gi and from ReadWriteOnce to ReadWriteMany
```
spec:
  resources:
    requests:
      storage: 20Gi
  accessModes:
  - ReadWriteMany
```
Saved and restarted the jupyter-web-app-deployment.
In the New Notebook web form, the "New volume" header is updated, but the form is still showing the original defaults.
<img width="825" alt="image" src="https://user-images.githubusercontent.com/36431265/201811048-0e5ed4b3-ca52-4564-bbfc-81ebdb463589.png">
Proceeding with creating the notebook server, without touching or changing anything in the form, creates the pods with default values from the configmap. Although, regular users will be confused by the selected options in the UI.

It looks like the form selection is hardcoded into the ui.
https://github.com/kubeflow/kubeflow/blob/master/components/crud-web-apps/jupyter/frontend/src/app/shared/utils/volumes/forms.ts#L79-L83

**What did you expect to happen:**
Changes in the configMap as default values would reflect in the UI.

**Environment:**

- Kubeflow version: build version v1.5.0
- Kubernetes platform: kubeadm/clusterAPI
- OS: RHEL 8.4

**Comment by kimwnasptd** [comment]
Looks like a small bug in how the form handles these values. Thank you for raising this @alexchrisramos!

/assign @kimwnasptd

**Comment by andreyvelich** [comment]
This was fixed in #7029
